deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -57,7 +57,7 @@ metadata:
     capabilities: Full Lifecycle
     categories: Storage
     containerImage: quay.io/ocs-dev/ocs-operator:4.5.0
-    createdAt: "2020-06-02 19:15:16"
+    createdAt: "2020-06-03 13:03:39"
```

Comment:
This shouldn't be needed. Try and remove it from the patch. (As @jarrpa requested.)

Reply:
Thanks @obnoxxx ... I have reverted those two files and ran `make verify-latest-csv` successfully.